moveit_configs_utils/default_configs/pilz_industrial_motion_planner_planning.yaml

```
@@ -0,0 +1,5 @@
+planning_plugin: pilz_industrial_motion_planner::CommandPlanner
```

Better use the specified plugin name. We've seen weird issues when mixing plugin names with class names (even though it shouldn't make a difference).
```suggestion
planning_plugin: pilz_industrial_motion_planner/CommandPlanner
```

---

Copied from [here](https://github.com/ros-planning/moveit2/blob/9e30bdc77d0a9386dfdc50766e782de8be155010/moveit_setup_assistant/templates/moveit_config_pkg_template/launch/pilz_industrial_motion_planner_planning_pipeline.launch.xml#L6)